e2e-tests/steps/explore/graphs-overview-steps.js

```diff
@@ -73,4 +77,16 @@ export class GraphsOverviewSteps {
     static clickOnDownloadAllButton() {
         this.getDownloadAllButtons().click();
     }
+
+    static getGraphsSearchInput() {
```

Comment:
I'd prefer selectors to be scoped where it makes sence, e.g. `this.getView().find('.search-graphs')` etc.

Reply:
OK, I’ll do it in the next MR.